Libs/MRML/DisplayableManager/vtkMRMLModelDisplayableManager.cxx

```diff
@@ -492,15 +502,15 @@ void vtkMRMLModelDisplayableManager::ProcessMRMLNodesEvents(vtkObject *caller,
       vtkMRMLDisplayableNode::SafeDownCast(caller);
     switch (event)
       {
-       case vtkMRMLDisplayableNode::DisplayModifiedEvent:
+      case vtkMRMLDisplayableNode::DisplayModifiedEvent:
          // don't go any further if the modified display node is not a model
-         if (!this->IsModelDisplayable(displayableNode) &&
-             !this->IsModelDisplayable(
-               reinterpret_cast<vtkMRMLDisplayNode*>(callData)))
-          {
-          requestRender = false;
-          break;
-          }
+        if (!this->IsModelDisplayable(displayableNode) &&
+            !this->IsModelDisplayable(
+              reinterpret_cast<vtkMRMLDisplayNode*>(callData)))
+         {
+         requestRender = false;
+         break;
+         }
```

Comment:
Needs one more space in indentation of brackets and content (10 spaces) - to be confirmed (hard to check on github)